src/components/custom/_eu-redirect-modal.tsx

```diff
@@ -85,7 +95,16 @@ const Cancel = styled.span`
     }
 `
 
-const EURedirect = ({ toggle, is_open, closeModal, to, target, rel, ref, aria_label }) => {
+const EURedirect = ({
+    toggle,
+    is_open,
+    closeModal,
+    to,
+    target,
+    rel,
+    ref,
+    aria_label,
```

Comment:
Let's add default value for optional props

```suggestion
    toggle,
    is_open,
    closeModal,
    to = '',
    target = '',
    rel = '',
    ref,
    aria_label = '',
```